deploy/Dockerfile

```diff
@@ -60,7 +61,7 @@ RUN GOARCH=$(cat /goarch) && (mkdir -p /go/src/github.com/chrismellard || true)
 RUN mkdir -p /kaniko/.docker
 
 COPY . .
-RUN make GOARCH=$(cat /goarch)
+RUN make GOARCH=$(cat /goarch.txt)
```

Comment:
There is only a `/goarch` file, none with a `.txt` suffix. This is also wrong in master already (https://github.com/GoogleContainerTools/kaniko/blob/master/deploy/Dockerfile#L53). So, kind of surprised to see this as a change here.